package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "dotenv-cli",
   "description": "A global executable to run applications with the ENV variables loaded by dotenv",
-  "version": "3.0.0",
+  "version": "4.0.0",
```

Comment:
It's considered bad practice to bump a version number when doing a PR to an open source project. 😄